app/api/routes/dependencies.py

```diff
@@ -18,21 +18,20 @@ async def get_current_user(token: str = Depends(oauth2_scheme)) -> TokenData:
     try:
         #Try to verify the token
         username = verify_token(token, credentials_exception)
-    except HTTPException:
-        raise credentials_exception
-    
-    try:
+
         #Fetch the user
         user = next((u for u in users_db.values() if str(u.id) == username), None)
         if user is None or not user.is_active:
             raise credentials_exception
+        
+        return TokenData(username=UUID(username))
+
```

Comment:
user_id???

Reply:
I've made some changes to the code, i hope this improves it for the better.